docs/atlascli/command/atlas-clusters-availableRegions-list.txt

```diff
@@ -0,0 +1,84 @@
+.. _atlas-clusters-availableRegions-list:
+
+====================================
+atlas clusters availableRegions list
+====================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+List available regions for your project.
```

Comment:
Just a question about what this means because all regions are "available" in that one could create a cluster in any region at any time, right? So does this mean "regions in which you have already have clusters deployed" or "regions in which you have not deployed a free cluster"? Or do we mean what regions are available based on restrictions by user's location?

Reply:
Ah ok, could we change it to say "List available regions in which you can create new deployments"? That would make it clear we mean available for new deployments and not available regions in which they have deployments already

Reply:
sure, on it! thank you!